java-manta-client-unshaded/src/main/java/com/joyent/manta/http/MantaConnectionFactory.java

```diff
@@ -87,19 +78,13 @@
  * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
  * @since 3.0.0
  */
-public class MantaConnectionFactory implements Closeable {
+public class MantaConnectionFactory implements Closeable, MBeanable {
```

Comment:
I like this refactoring. It reduces the overall responsibility for the`MantaConnectionFactory` class.